docs/source/languages/orquesta.rst

```diff
@@ -101,6 +101,9 @@ transition will be represented as an outbound edge of the current task. When the
 specified in ``when`` of the transition is met, the next set of tasks under ``do`` will be invoked.
 If there are no more outbound edges identified, then the workflow execution is complete.
 
+Orquesta workflows are designed to fail fast. If a task fails which has no transition for that
+failure condition, then the workflow will stop as soon as any in-progress tasks are completed.
```

Comment:
Probably should have some language in here about branches, maybe:

This means that if there are multiple parallel branches and a task in one branch `fails`, then Orquesta will wait for all currently running tasks in the other branches to finish and terminate the workflow. In other words, once one task in one branch `fails`, no other tasks will be queued or scheduled for that entire workflow no matter what branch they are executing on.


Thoughts?

Reply:
Yep, the more details the better. Maybe even give an example...

Reply:
I originally was going to use branches and state that it will fail as soon as one branch fails, but then I noticed that we didn't really mention branches in the description of the workflow in the earlier page. So it would introduce a new concept.

Reply:
@nmaludy Happy to add on that suggestion of yours to the end as to me it's about branches, but I was just a bit surprised it wasn't mentioned earlier - so didn't want to confuse.